zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -443,7 +456,11 @@ function ProjectDetails(props) {
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
         >
-          <img className={classes.enlargedImageStyle} src={enlarged_image_url} alt={`${project.title}`} />
+          <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', width: '80%', maxWidth: '600px', }} >
```

Comment:
Why not push these into `projectDetailsStyles.js`?

Reply:
Alright @coderatomy , thank you for the suggestions, I think they are really good, and I missed those. I'll work on them today, and push a commit to this same PR..